data/mil2525d/utilities/military-features-utilities/template-gdb-toolbox/scripts/CreateTemplateGDB/CreateTemplateGDB.py

```diff
@@ -0,0 +1,197 @@
+# ----------------------------------------------------------------------------------
+# Copyright 2015 Esri
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+# CreateTemplateGDB.py
+#
+# Description: Creates an empty file geodatabase from the contents of
+# an input CSV file.  
+#
+# Adds a feature dataset and feature classes based on a supplied schema.  
+#
+# Adds fields to all those feature classes based on their supplied schemas.
+#
+# Requirements: ArcGIS Desktop
+# ----------------------------------------------------------------------------------
+
+import arcpy, os, sys, traceback
+import AddFieldsFromSchema
+import csv as csv
+import time
+
+from arcpy import env
+from os import path
+
+def geometryTypeLookup(jmsmlGeometryType):
+    
+    # Converts a JMSML symbol type into an Esri GIS geometry type
+    
+    switcher = {
+                "Point": "POINT",
+                "Line": "POLYLINE",
+                "Area": "POLYGON",
+                }
+    return switcher.get(jmsmlGeometryType, "NONE")
+
+def createVersionsTable(schemasFolder, geodatabase):
+    
+    # Creates a version table using the contents of the specified CSV file
+    # and today's date.  Inserts the table into the specified geodatabase
+    
+    try:
+        arcpy.AddMessage("Starting: CreateVersionsTable")
+        
+        currentPath = os.path.dirname(__file__) 
+        versionFile = os.path.normpath(os.path.join(currentPath, "../../../../style-utilities/merge-stylx-utilities/versions.csv"))
```

Comment:
I didn't realize these two items (stylx and gdb) had the same version info - so good call in using the same version source file